Northstar.Client/mod/resource/ui/menus/modlist.menu

```diff
@@ -407,11 +407,11 @@ resource/ui/menus/modlist.menu
 			
 			labelText	"#CLEAR_FILTERS"
 			wide		100
-			xpos		-15
-			ypos		-55
+			xpos		0
```

Comment:
What does this change for the mod list button?
I don't see any change with and without the PR:

##### Without

![ui-mod-main](https://github.com/user-attachments/assets/44b4d60f-bb95-40df-b82b-632b4b3f4791)

##### With

![ui-mod-pr](https://github.com/user-attachments/assets/65acbb69-3ca4-4342-9980-708e64e3377a)